bpdm-pool/src/test/kotlin/org/eclipse/tractusx/bpdm/pool/util/WebTestClientExtensions.kt

```diff
@@ -35,4 +35,3 @@ inline fun <reified T : Any> WebTestClient.invokeGetEndpoint(path: String, varar
         .responseBody
         .blockFirst()!!
 }
-
```

Comment:
Unrelated to the feature. Please revert back for more clarity

Reply:
I assume there must be some automatic optimization feature on your end then because it clearly shows that its different from the current state of main